cppcache/integration-test/testThinClientPdxTests.cpp

```diff
@@ -2346,7 +2346,7 @@ DUNIT_TASK_DEFINITION(CLIENT1, generateJavaPdxType)
 
     ResultCollectorPtr collector = funcExec->withArgs(args)
                                        ->withFilter(routingObj)
-                                       ->execute("ComparePdxTypes", true);
```

Comment:
We removed some deprecated parameters months ago where this would have matched a different method with defaults. Now it’s closest match was an implicit conversation to int. Fixed it.

Reply:
Ah, now that rings a bell.